test/onnx/onnx_test.cpp

```diff
@@ -4233,7 +4233,8 @@ TEST_CASE(round_test)
     EXPECT(p == prog);
 }
 
-TEST_CASE(scatter_test)
+// the ScatterElements op has 3 reduction modes, which map to separate reference ops
+TEST_CASE(scatter_add_test)
```

Comment:
Like I mentioned in test/ref_ops_test.cpp, you can create a function that creates a program where the reduction mode is passed in as a parameter.

Reply:
changed